### PR TITLE
Restore scrolling / dragging

### LINKED ISSUE #895: Restore tab scrolling/dragging
__Tasks__

* [x] Tabs can be re-ordered via drag and drop
* [x] Tabs are shown in one line and can be scrolled via arrow and wheel

Hints:

* try reusing bpmn.io tab scrolling and dragging util libraries

### PR DESCRIPTION
**Pull request by philippfromme**
* select tabs through scrolling
* move tabs through dragging

Closes #895

### CODE REVIEW

**Comment by philippfromme** [comment]
Showing the empty tab when tabs are overflowing is extremely difficult without adjustments in the [scroll-tabs](https://github.com/bpmn-io/scroll-tabs) library.

**Comment by philippfromme** [comment]
@barmac Ready for review.

**Comment by philippfromme** [comment]
Regarding the empty tab I've created a seperate issue: https://github.com/camunda/camunda-modeler/issues/968

File: client/src/app/primitives/TabLinks.js [hunk 1]
```diff
@@ -4,10 +4,59 @@ import classNames from 'classnames';
...
+    }
+  }
+
+  handleScroll = (node) => {
+    const {
+      onSelect,
+      tabs
+    } = this.props;
+
+    const tab = find(tabs, matchPattern({ id: node.dataset.tabId }));
```
Comment (barmac) [inline client/src/app/primitives/TabLinks.js]: `const tab = tabs.find(tab => tab.id === node.dataset.tabId);`

**Comment by barmac** [comment]
Note: use throttling for tab scrolling

File: client/src/app/primitives/TabLinks.js (hunk 1 above)
Comment (philippfromme) [reply client/src/app/primitives/TabLinks.js]: Done. 👍

**Comment by philippfromme** [comment]
> * [ ]  use throttling for tab scrolling

Throttling won't fix this issue. The problem is that delta is different depending on how you scroll (mouse wheel, touchpad). We had the same issue in [diagram-js](https://github.com/bpmn-io/diagram-js/blob/master/lib/navigation/zoomscroll/ZoomScroll.js).

**Comment by barmac** [comment]
> > * [ ]   use throttling for tab scrolling
> 
> Throttling won't fix this issue. The problem is that delta is different depending on how you scroll (mouse wheel, touchpad). We had the same issue in [diagram-js](https://github.com/bpmn-io/diagram-js/blob/master/lib/navigation/zoomscroll/ZoomScroll.js).
> 
> I've created a seperate issue for that: #981

It will, see my answer in the linked issue. However, it's okay to handle it separately.

**Comment by barmac** [comment]
Rebased on top of `next` and removed unused imports.

**Pull request merged**